docs/snippets/dataStorage/datastorage_interface.cpp

```diff
@@ -0,0 +1,44 @@
+#include <alpaka/alpaka.hpp>
+
+int* ptr;
+
+alpaka::LinearizedIdxGenerator otherLinearizedIdxGenerator{alpaka::Vec{0}};
+alpaka::MdSpan otherMdspan{ptr, alpaka::Vec{0}, alpaka::Vec{0}};
+alpaka::View otherView{alpaka::api::host, ptr, alpaka::Vec{0}, alpaka::Vec{0}};
+alpaka::onHost::SharedBuffer otherSharedBuffer = alpaka::onHost::allocHost<int>(alpaka::Vec{0});
+
+// BEGIN-DATASTORAGE-interface
+void func1(alpaka::concepts::IDataSource auto input)
+{
+}
+
+void func2(alpaka::concepts::IView auto input)
+{
+}
+
+int main()
+{
+    // fulfil the IDataSource interface, but not the IMdSpan interface
```

Comment:
here and in the next lines too
```suggestion
    // fulfill the IDataSource interface, but not the IMdSpan interface
```